completions/openssl

```diff
@@ -110,6 +111,11 @@ _comp_cmd_openssl()
                 dgst | req | x509)
                     _comp_compgen -a -i openssl digests "$1"
                     ;;
+                enc)
+                    # The -list option was added in openssl 1.1.1e, so we ignore the error here.
+                    # We do get some of the ciphers "by accident", since some are detected in the help output.
+                    _comp_compgen -a split -- "$("$1" enc -list 2>/dev/null || true | sed '/Supported ciphers:/d')"
```

Comment:
`cmd1 || cmd2 | cmd3` would be interpreted as `cmd1 || { cmd2 | cmd3; }`, which I think is not the intention.

```suggestion
                    _comp_compgen -a split -- "$({ "$1" enc -list 2>/dev/null || true; } | sed '/Supported ciphers:/d')"
```

Reply:
Do we actually need to insert `true` here? In what situation, would the non-zero exit status cause an issue? I thought that `-o pipefail` might cause a signal in `sed`, but it doesn't seem to be the case. `-o errexit` doesn't seem to cause an issue either. The exit status of the command substitution would anyway be ignored.

Reply:
Yeah you're correct. Added `command`

Reply:
Thanks!